lib/charms/mongodb/v1/helpers.py

```diff
@@ -46,19 +46,27 @@
 
 DATA_DIR = "/var/lib/mongodb"
 LOG_DIR = "/var/log/mongodb"
-LOG_TO_SYSLOG = True
 CONF_DIR = "/etc/mongod"
 MONGODB_LOG_FILENAME = "mongodb.log"
 logger = logging.getLogger(__name__)
 
 
 def _get_logging_options(snap_install: bool) -> str:
-    # TODO sending logs to syslog until we have a separate mount point for logs
```

Comment:
Could you add a short docstring to this function?

Reply:
➕